wiki/Help_centre/es.md

```diff
@@ -0,0 +1,79 @@
+5. Haz clic en el botón de reinicio que aparece en la parte inferior de la pantalla si realizaste algún cambio en el flujo de lanzamiento para aplicarlos.
+6. Ve al menú `Opciones` nuevamente y haz clic en `Abrir carpeta de osu!`.
+7. Encuentra el directorio `Logs` en la ventana que se abre.
+8. Selecciona el archivo de registro necesario (el representante de soporte te indicará cuál) y adjúntalo a tu ticket de soporte o publicación en el foro.
+
+### Visor de eventos {id=event-viewer}
+
+**El Visor de eventos es un componente integrado de Windows que se puede utilizar para encontrar registros de bloqueo cuando osu! no te proporciona ninguno. Estos registros de bloqueo se pueden usar para determinar qué está causando tu problema.**
+
+Cuando tengas problemas con osu! bloqueándose y osu! no te proporcione un registro de bloqueo, el único lugar para encontrar uno es a través del Visor de eventos.
+
+Si un miembro del equipo de soporte te ha pedido que encuentres un registro de bloqueo en el Visor de eventos, así es cómo hacerlo:
+
+1. Después de que osu! se haya bloqueado, presiona `Win` + `R` para abrir la ventana de ejecutar.
+2. En la ventana de ejecutar, escribe `eventvwr` y presiona `Enter`. Esto abrirá el Visor de eventos.
+3. En el Visor de eventos, a la izquierda, haz clic en `Registros de Windows` y luego en `Aplicación`.
+4. A la derecha, haz clic en `Filtrar registro actual`.
+5. En la ventana de filtro que se abre, asegúrate de que la casilla `Error` esté marcada y haz clic en `Aceptar`.
+6. Presiona `Ctrl` + `F` y escribe osu! en el cuadro de búsqueda. Encontrará el primer registro de bloqueo de osu!.
+7. Ve a la pestaña `Detalles`, expande `Sistema` y `Datos del evento` haciendo clic en cada uno de ellos.
+8. Copia el texto de allí y pégalo en tu ticket de soporte o publicación en el foro.
+
+### Gráfico de tiempo de fotograma {id=frame-time-graph}
+
+**El Gráfico de tiempo de fotograma es una función de osu! que nos puede ayudar a recopilar información más útil sobre los problemas de rendimiento que puedes estar experimentando.**
+
+Cuando te enfrentes a problemas de rendimiento complicados en osu!, el Gráfico de tiempo de fotograma es una buena herramienta para ayudarnos a diagnosticar el problema y ayudarte a encontrar una solución.
+
+Si un miembro del equipo de soporte te ha pedido que tomes un video o captura de pantalla con el Gráfico de tiempo de fotograma abierto, así es cómo hacerlo:
+
+1. Abre osu!.
+2. Presiona `Ctrl` + `F11` para abrir el Gráfico de tiempo de fotogramas.
+3. Provoca o espera a que ocurra el problema de rendimiento.
+4. Toma una captura de pantalla presionando `Shift` + `F12`. Esto cargará la captura de pantalla en los servidores de osu! y abrirá la captura de pantalla en una pestaña del navegador web.
+5. Copia y pega el enlace proporcionado en tu ticket de soporte o publicación en el foro.
+6. Cierra el Gráfico de tiempo de fotograma de la misma manera en que lo abriste, presionando `Ctrl` + `F11`.
```

Comment:
```suggestion
6. Cierra el gráfico de tiempo entre fotogramas de la misma manera en que lo abriste, presionando `Ctrl` + `F11`.
```

Reply:
Thanks for pointing out all the translation mistakes!